src/SourceBrowser.Site/Views/Shared/_BrowseLayout.cshtml

```diff
@@ -19,12 +19,10 @@
             <a class="element visible-on-wide-screen after-divider" href="/Browse/">Browse</a>
             <a class="element visible-on-wide-screen" href="/Upload/">Upload</a>
             <div class="input-element place-right" id="search-form">
-                <form>
-                    <div class="input-control text">
-                        <input type="text" id="search-box" placeholder="Search...">
-                        <button type="submit" id="search-button"></button>
-                    </div>
-                </form>
+                <div class="input-control text">
+                    <input type="text" id="search-box" placeholder="Search...">
+                    <button type="submit" id="search-button"></button>
```

Comment:
This probably shouldn't be of type submit if it's not in the form anymore. I think you can probably just get rid of the type.